frontend/src/scenes/settings/organization/InviteModal.tsx

```diff
@@ -38,9 +40,160 @@ export function EmailUnavailableMessage(): JSX.Element {
+                        </span>
+                    }
+                >
+                    <IconInfo className="text-muted-alt" />
+                </Tooltip>
+                {availableProjectsToShow.length > 0 && (
+                    <LemonSelect
+                        icon={<IconPlus />}
+                        className="bg-bg-light"
+                        placeholder="Add project"
+                        options={availableProjectsToShow.map((project: any) => ({
+                            value: project.id,
+                            label: project.name,
+                        }))}
+                        onChange={(projectId) => {
+                            if (projectId) {
+                                addProjectAccess(Number(projectId), AccessControlLevel.Member)
+                            }
+                        }}
+                    />
+                )}
+            </div>
+
+            {isOrgLevelAdminOrOwner && selectedProjects.length > 0 && (
+                <LemonBanner type="warning" className="text-xs">
+                    This user will have{' '}
+                    <span className="font-bold italic">{OrganizationMembershipLevel[invite.level].toLowerCase()}</span>{' '}
+                    access on the organization level, which will override any project-specific access controls.
+                </LemonBanner>
+            )}
+
+            {selectedProjects.length > 0 && (
+                <div className="space-y-2">
+                    {selectedProjects.map((access) => {
+                        const project = availableProjects.find((p: any) => p.id === access.id)
+                        if (!project) {
+                            return null
+                        }
+
+                        const defaultLevel = projectAccessControls[project.id]?.access_level
+                        const isLowerThanDefault = defaultLevel === 'admin' && access.level === 'member'
+
+                        return (
+                            <div key={access.id} className="space-y-2">
+                                <div className="p-2 bg-bg-light rounded border">
+                                    {isLowerThanDefault && (
+                                        <div className="mb-2">
+                                            <LemonBanner type="warning" className="text-xs">
+                                                <strong>{project.name}</strong> has a default access level of{' '}
+                                                <span className="font-bold italic">{defaultLevel}</span>. Since you
+                                                selected <span className="font-bold italic">{access.level}</span> (which
+                                                is lower), the user will actually get{' '}
+                                                <span className="font-bold italic">{defaultLevel}</span> access.
+                                            </LemonBanner>
```

Comment:
IMO this is kind of silly, shouldn't we prevent them from selecting a membership level lower than the default? Show it in the level dropdown, but not selectable, and a tooltip saying why they can't select it.